protocol-testing/src/test_runner.rs

```diff
@@ -423,7 +423,8 @@ impl TestRunner {
                             execution_data,
                             &block,
                             &config.protocol_system,
-                            &[], // No skip filters for live testing
+                            &[],   // No skip filters for live testing
+                            false, // is_current_block - live testing uses historical blocks
```

Comment:
God I haven't run this with the full range yet D: Can I do that in a separate PR please